package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@thunderbirdops/services-ui",
-  "version": "0.6.2",
+  "version": "0.6.1",
```

Comment:
Any reason for the version downgrade?

Reply:
nooo, merge error, I'll change it
good catch!